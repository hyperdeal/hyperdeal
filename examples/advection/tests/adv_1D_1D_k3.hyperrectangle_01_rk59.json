{
    "General": {
        "DimX": "1",
        "DimV": "1",
        "DegreeX": "3",
        "DegreeV": "3",
        "PartitionX": "1",
        "PartitionV": "1",
        "UseVirtualTopology": "true",
        "UseSharedMemory": "true",
        "Verbose": "true",
        "Case": "hyperrectangle"
    },
    "SpatialDiscretization": {
        "MappingX": "1",
        "MappingV": "1",
        "DoCollocation": "false"
    },
    "Triangulation": {
        "Type": "fullydistributed",
        "OutputGrid": "false"
    },
    "TemporalDiscretization": {
        "TimeStep": "0.1",
        "StartTime": "0",
        "FinalTime": "20",
        "MaxTimeStepNumber": "100000000",
        "RKType": "rk59",
        "CFLNumber": "0.3",
        "DiagnosticsEnabled": "true",
        "DiagnosticsTick": "0.1",
        "PerformanceLogAllCalls": "false",
        "DiagnosticsWarmUpIterations": "0"
    },
    "AdvectionOperation": {
        "SkewFactor": "0.0"
    },
    "Matrixfree": {
        "GhostFaces": "true",
        "DoBuffering": "false",
        "UseECL": "true"
    },
    "Case": {
        "NRefinementsX": "0",
        "NRefinementsV": "0",
        "PeriodicX": "true",
        "PeriodicV": "true",
        "NSubdivisionsX": {
            "X": "4",
            "Y": "4"
        },
        "NSubdivisionsV": {
            "X": "4",
            "Y": "4"
        }
    }
}
