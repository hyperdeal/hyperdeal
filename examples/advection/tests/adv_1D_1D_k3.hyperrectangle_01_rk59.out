   Time: 0.000e+00, norm:  9.9038504150e-01, error:  3.0671866339e-02
   Time: 1.154e-01, norm:  9.8948430541e-01, error:  3.9403842974e-02
   Time: 2.020e-01, norm:  9.8832696447e-01, error:  2.5769553902e-02
   Time: 3.175e-01, norm:  9.8816709256e-01, error:  1.5665563726e-02
   Time: 4.040e-01, norm:  9.8781974743e-01, error:  1.9381905690e-02
   Time: 5.195e-01, norm:  9.8624371360e-01, error:  3.1047745377e-02
   Time: 6.061e-01, norm:  9.8477460556e-01, error:  3.3709063498e-02
   Time: 7.215e-01, norm:  9.8374853320e-01, error:  2.7293937691e-02
   Time: 8.081e-01, norm:  9.8364020651e-01, error:  2.0912674890e-02
   Time: 9.235e-01, norm:  9.8305040310e-01, error:  2.6148333437e-02
   Time: 1.010e+00, norm:  9.8175789008e-01, error:  3.4522625951e-02
   Time: 1.126e+00, norm:  9.7977984211e-01, error:  3.8511060672e-02
   Time: 1.212e+00, norm:  9.7898099662e-01, error:  3.5167125042e-02
   Time: 1.328e+00, norm:  9.7868945564e-01, error:  3.0759370599e-02
   Time: 1.414e+00, norm:  9.7811875189e-01, error:  3.4681242784e-02
   Time: 1.501e+00, norm:  9.7679243391e-01, error:  4.1706385572e-02
   Time: 1.616e+00, norm:  9.7465127212e-01, error:  4.6157184915e-02
   Time: 1.703e+00, norm:  9.7367921398e-01, error:  4.3972590488e-02
   Time: 1.818e+00, norm:  9.7325208923e-01, error:  3.9963884053e-02
   Time: 1.905e+00, norm:  9.7268775503e-01, error:  4.2152617212e-02
   Time: 2.020e+00, norm:  9.7083684115e-01, error:  4.9346404380e-02
   Time: 2.107e+00, norm:  9.6920419714e-01, error:  5.1855922759e-02
   Time: 2.222e+00, norm:  9.6797534479e-01, error:  4.8818895708e-02
   Time: 2.309e+00, norm:  9.6771677509e-01, error:  4.5662702109e-02
   Time: 2.424e+00, norm:  9.6695183983e-01, error:  4.7733862781e-02
   Time: 2.511e+00, norm:  9.6557734612e-01, error:  5.2356330283e-02
   Time: 2.626e+00, norm:  9.6356890042e-01, error:  5.4689415594e-02
   Time: 2.713e+00, norm:  9.6278724213e-01, error:  5.1918818490e-02
   Time: 2.828e+00, norm:  9.6254854551e-01, error:  4.7638995675e-02
   Time: 2.915e+00, norm:  9.6205104659e-01, error:  4.8906753380e-02
   Time: 3.001e+00, norm:  9.6084579920e-01, error:  5.3006214434e-02
   Time: 3.117e+00, norm:  9.5894117700e-01, error:  5.5602444822e-02
   Time: 3.203e+00, norm:  9.5817662818e-01, error:  5.3106671062e-02
   Time: 3.319e+00, norm:  9.5802872355e-01, error:  4.8457259478e-02
   Time: 3.405e+00, norm:  9.5767658126e-01, error:  4.9134810572e-02
   Time: 3.521e+00, norm:  9.5613447124e-01, error:  5.4489102953e-02
   Time: 3.608e+00, norm:  9.5475055458e-01, error:  5.6536484849e-02
   Time: 3.723e+00, norm:  9.5383623858e-01, error:  5.3670772201e-02
   Time: 3.810e+00, norm:  9.5377572252e-01, error:  5.0652878159e-02
   Time: 3.925e+00, norm:  9.5322772626e-01, error:  5.2382303455e-02
   Time: 4.012e+00, norm:  9.5199808720e-01, error:  5.6887403558e-02
   Time: 4.127e+00, norm:  9.5015995684e-01, error:  5.9955001066e-02
   Time: 4.214e+00, norm:  9.4946596902e-01, error:  5.8350760232e-02
   Time: 4.300e+00, norm:  9.4931234641e-01, error:  5.6020159095e-02
   Time: 4.416e+00, norm:  9.4874141987e-01, error:  5.7730587314e-02
   Time: 4.502e+00, norm:  9.4748804439e-01, error:  6.2249572478e-02
   Time: 4.618e+00, norm:  9.4549369821e-01, error:  6.6150289156e-02
   Time: 4.704e+00, norm:  9.4462764366e-01, error:  6.5451774739e-02
   Time: 4.820e+00, norm:  9.4427792534e-01, error:  6.3420409983e-02
   Time: 4.906e+00, norm:  9.4373739653e-01, error:  6.4979451535e-02
   Time: 5.022e+00, norm:  9.4193782512e-01, error:  7.0457287843e-02
   Time: 5.108e+00, norm:  9.4036944770e-01, error:  7.3154392753e-02
   Time: 5.224e+00, norm:  9.3920253435e-01, error:  7.2362695552e-02
   Time: 5.310e+00, norm:  9.3893681331e-01, error:  7.0897833384e-02
   Time: 5.426e+00, norm:  9.3811899505e-01, error:  7.2716515618e-02
   Time: 5.512e+00, norm:  9.3672062295e-01, error:  7.6385999627e-02
   Time: 5.628e+00, norm:  9.3471699008e-01, error:  7.9229812773e-02
   Time: 5.714e+00, norm:  9.3392684322e-01, error:  7.8283420099e-02
   Time: 5.801e+00, norm:  9.3368893288e-01, error:  7.6503885920e-02
   Time: 5.916e+00, norm:  9.3304180965e-01, error:  7.7272633142e-02
   Time: 6.003e+00, norm:  9.3178441902e-01, error:  8.0312145968e-02
   Time: 6.118e+00, norm:  9.2986460776e-01, error:  8.2991595985e-02
   Time: 6.205e+00, norm:  9.2909271057e-01, error:  8.2072052736e-02
   Time: 6.320e+00, norm:  9.2889017662e-01, error:  7.9579322826e-02
   Time: 6.407e+00, norm:  9.2848239315e-01, error:  7.9979488282e-02
   Time: 6.522e+00, norm:  9.2691625478e-01, error:  8.3552102671e-02
   Time: 6.609e+00, norm:  9.2556385837e-01, error:  8.5346710869e-02
   Time: 6.724e+00, norm:  9.2470049256e-01, error:  8.4031821275e-02
   Time: 6.811e+00, norm:  9.2465050862e-01, error:  8.2153439238e-02
   Time: 6.926e+00, norm:  9.2410848421e-01, error:  8.2893080889e-02
   Time: 7.013e+00, norm:  9.2292324541e-01, error:  8.5740100349e-02
   Time: 7.128e+00, norm:  9.2120699241e-01, error:  8.8142480849e-02
   Time: 7.215e+00, norm:  9.2060828276e-01, error:  8.7308826500e-02
   Time: 7.302e+00, norm:  9.2052149265e-01, error:  8.5715863322e-02
   Time: 7.417e+00, norm:  9.2001821676e-01, error:  8.6456240656e-02
   Time: 7.504e+00, norm:  9.1884389508e-01, error:  8.9432165867e-02
   Time: 7.619e+00, norm:  9.1700773794e-01, error:  9.2525800956e-02
   Time: 7.706e+00, norm:  9.1625892564e-01, error:  9.2356046762e-02
   Time: 7.821e+00, norm:  9.1600915046e-01, error:  9.1007132861e-02
   Time: 7.908e+00, norm:  9.1551663934e-01, error:  9.1982180831e-02
   Time: 8.023e+00, norm:  9.1380803853e-01, error:  9.6073559462e-02
   Time: 8.110e+00, norm:  9.1233627396e-01, error:  9.8518043934e-02
   Time: 8.225e+00, norm:  9.1127678905e-01, error:  9.8602252606e-02
   Time: 8.312e+00, norm:  9.1104095551e-01, error:  9.7802378360e-02
   Time: 8.427e+00, norm:  9.1022304284e-01, error:  9.9306055939e-02
   Time: 8.514e+00, norm:  9.0883962543e-01, error:  1.0234753457e-01
   Time: 8.600e+00, norm:  9.0730366628e-01, error:  1.0487570728e-01
   Time: 8.716e+00, norm:  9.0611611291e-01, error:  1.0522676744e-01
   Time: 8.802e+00, norm:  9.0585425437e-01, error:  1.0433746275e-01
   Time: 8.918e+00, norm:  9.0513389294e-01, error:  1.0522563555e-01
   Time: 9.004e+00, norm:  9.0383785908e-01, error:  1.0783335502e-01
   Time: 9.120e+00, norm:  9.0190821017e-01, error:  1.1063979728e-01
   Time: 9.206e+00, norm:  9.0112371352e-01, error:  1.1057218983e-01
   Time: 9.322e+00, norm:  9.0085070585e-01, error:  1.0921258283e-01
   Time: 9.408e+00, norm:  9.0037017918e-01, error:  1.0965094552e-01
   Time: 9.524e+00, norm:  8.9874882010e-01, error:  1.1261686767e-01
   Time: 9.610e+00, norm:  8.9739712638e-01, error:  1.1441280725e-01
   Time: 9.726e+00, norm:  8.9653586424e-01, error:  1.1400117981e-01
   Time: 9.812e+00, norm:  8.9645741908e-01, error:  1.1276703839e-01
   Time: 9.928e+00, norm:  8.9586954508e-01, error:  1.1324077087e-01
   Time: 1.001e+01, norm:  8.9468967099e-01, error:  1.1539608903e-01
   Time: 1.010e+01, norm:  8.9338149082e-01, error:  1.1726000572e-01
   Time: 1.022e+01, norm:  8.9249963645e-01, error:  1.1711946045e-01
   Time: 1.030e+01, norm:  8.9244440433e-01, error:  1.1592010021e-01
   Time: 1.042e+01, norm:  8.9197079293e-01, error:  1.1619607724e-01
   Time: 1.051e+01, norm:  8.9085344740e-01, error:  1.1832801817e-01
   Time: 1.062e+01, norm:  8.8915249174e-01, error:  1.2085158749e-01
   Time: 1.071e+01, norm:  8.8850904555e-01, error:  1.2086947423e-01
   Time: 1.082e+01, norm:  8.8835426485e-01, error:  1.1974886032e-01
   Time: 1.091e+01, norm:  8.8791567049e-01, error:  1.2028322695e-01
   Time: 1.102e+01, norm:  8.8631630338e-01, error:  1.2337382710e-01
   Time: 1.111e+01, norm:  8.8495993329e-01, error:  1.2547817924e-01
   Time: 1.123e+01, norm:  8.8403892590e-01, error:  1.2583241221e-01
   Time: 1.131e+01, norm:  8.8386224382e-01, error:  1.2524865516e-01
   Time: 1.143e+01, norm:  8.8309133536e-01, error:  1.2636975880e-01
   Time: 1.152e+01, norm:  8.8176135861e-01, error:  1.2889257206e-01
   Time: 1.160e+01, norm:  8.8030095208e-01, error:  1.3123441607e-01
   Time: 1.172e+01, norm:  8.7919528688e-01, error:  1.3202241522e-01
   Time: 1.180e+01, norm:  8.7894573825e-01, error:  1.3154684071e-01
   Time: 1.192e+01, norm:  8.7819965073e-01, error:  1.3242170052e-01
   Time: 1.201e+01, norm:  8.7689717670e-01, error:  1.3474596499e-01
   Time: 1.212e+01, norm:  8.7499149896e-01, error:  1.3759349680e-01
   Time: 1.221e+01, norm:  8.7421240573e-01, error:  1.3801513447e-01
   Time: 1.232e+01, norm:  8.7388536747e-01, error:  1.3733394350e-01
   Time: 1.241e+01, norm:  8.7333919760e-01, error:  1.3785970161e-01
   Time: 1.253e+01, norm:  8.7166232974e-01, error:  1.4059572198e-01
   Time: 1.261e+01, norm:  8.7030196186e-01, error:  1.4247657413e-01
   Time: 1.273e+01, norm:  8.6941934684e-01, error:  1.4270412513e-01
   Time: 1.281e+01, norm:  8.6928939677e-01, error:  1.4194103276e-01
   Time: 1.290e+01, norm:  8.6889531790e-01, error:  1.4205222942e-01
   Time: 1.302e+01, norm:  8.6741401371e-01, error:  1.4431828648e-01
   Time: 1.310e+01, norm:  8.6611777608e-01, error:  1.4610465465e-01
   Time: 1.322e+01, norm:  8.6525982137e-01, error:  1.4635985712e-01
   Time: 1.330e+01, norm:  8.6519599794e-01, error:  1.4547994826e-01
   Time: 1.342e+01, norm:  8.6470167137e-01, error:  1.4560347394e-01
   Time: 1.351e+01, norm:  8.6360575278e-01, error:  1.4729954187e-01
   Time: 1.362e+01, norm:  8.6199442914e-01, error:  1.4951360260e-01
   Time: 1.371e+01, norm:  8.6142482554e-01, error:  1.4965238627e-01
   Time: 1.382e+01, norm:  8.6133116094e-01, error:  1.4866550916e-01
   Time: 1.391e+01, norm:  8.6092920807e-01, error:  1.4891138557e-01
   Time: 1.403e+01, norm:  8.5942561931e-01, error:  1.5130930030e-01
   Time: 1.411e+01, norm:  8.5817996351e-01, error:  1.5311144177e-01
   Time: 1.423e+01, norm:  8.5739888476e-01, error:  1.5350499103e-01
   Time: 1.431e+01, norm:  8.5729036960e-01, error:  1.5295415574e-01
   Time: 1.440e+01, norm:  8.5687760987e-01, error:  1.5324859837e-01
   Time: 1.452e+01, norm:  8.5533778909e-01, error:  1.5574658771e-01
   Time: 1.460e+01, norm:  8.5398014348e-01, error:  1.5782886810e-01
   Time: 1.472e+01, norm:  8.5299631440e-01, error:  1.5871858149e-01
   Time: 1.481e+01, norm:  8.5279292124e-01, error:  1.5838480806e-01
   Time: 1.492e+01, norm:  8.5207137044e-01, error:  1.5910523714e-01
   Time: 1.501e+01, norm:  8.5080117056e-01, error:  1.6114800655e-01
   Time: 1.512e+01, norm:  8.4896756600e-01, error:  1.6390019985e-01
   Time: 1.521e+01, norm:  8.4822641014e-01, error:  1.6456259815e-01
   Time: 1.532e+01, norm:  8.4788546404e-01, error:  1.6424818766e-01
   Time: 1.541e+01, norm:  8.4730109830e-01, error:  1.6480764450e-01
   Time: 1.553e+01, norm:  8.4559796988e-01, error:  1.6739482490e-01
   Time: 1.561e+01, norm:  8.4424278868e-01, error:  1.6934024569e-01
   Time: 1.573e+01, norm:  8.4334671308e-01, error:  1.7001233906e-01
   Time: 1.582e+01, norm:  8.4316515444e-01, error:  1.6959797874e-01
   Time: 1.590e+01, norm:  8.4270236529e-01, error:  1.6983165761e-01
   Time: 1.602e+01, norm:  8.4116242862e-01, error:  1.7199573482e-01
   Time: 1.610e+01, norm:  8.3985861390e-01, error:  1.7381527927e-01
   Time: 1.622e+01, norm:  8.3898910431e-01, error:  1.7444030567e-01
   Time: 1.631e+01, norm:  8.3888600023e-01, error:  1.7386132700e-01
   Time: 1.642e+01, norm:  8.3832740306e-01, error:  1.7402504074e-01
   Time: 1.651e+01, norm:  8.3721788434e-01, error:  1.7553575263e-01
   Time: 1.662e+01, norm:  8.3564545120e-01, error:  1.7765236331e-01
   Time: 1.671e+01, norm:  8.3511023077e-01, error:  1.7793840001e-01
   Time: 1.683e+01, norm:  8.3502812474e-01, error:  1.7712369343e-01
   Time: 1.691e+01, norm:  8.3463014692e-01, error:  1.7723296062e-01
   Time: 1.703e+01, norm:  8.3318424283e-01, error:  1.7920868846e-01
   Time: 1.711e+01, norm:  8.3202386807e-01, error:  1.8080865295e-01
   Time: 1.720e+01, norm:  8.3142061363e-01, error:  1.8130502824e-01
   Time: 1.732e+01, norm:  8.3130053269e-01, error:  1.8065463764e-01
   Time: 1.740e+01, norm:  8.3093285406e-01, error:  1.8074252999e-01
   Time: 1.752e+01, norm:  8.2949180665e-01, error:  1.8274677106e-01
   Time: 1.760e+01, norm:  8.2824363707e-01, error:  1.8456619007e-01
   Time: 1.772e+01, norm:  8.2739746073e-01, error:  1.8540029974e-01
   Time: 1.781e+01, norm:  8.2725892267e-01, error:  1.8506678675e-01
   Time: 1.792e+01, norm:  8.2659664493e-01, error:  1.8554686578e-01
   Time: 1.801e+01, norm:  8.2538973832e-01, error:  1.8728847350e-01
   Time: 1.812e+01, norm:  8.2367173251e-01, error:  1.8982701958e-01
   Time: 1.821e+01, norm:  8.2300195411e-01, error:  1.9055085974e-01
   Time: 1.833e+01, norm:  8.2269411835e-01, error:  1.9036573326e-01
   Time: 1.841e+01, norm:  8.2210827702e-01, error:  1.9087026680e-01
   Time: 1.853e+01, norm:  8.2042554057e-01, error:  1.9326655877e-01
   Time: 1.861e+01, norm:  8.1910579796e-01, error:  1.9520061026e-01
   Time: 1.870e+01, norm:  8.1834788584e-01, error:  1.9608775679e-01
   Time: 1.882e+01, norm:  8.1801747771e-01, error:  1.9593236654e-01
   Time: 1.890e+01, norm:  8.1750356090e-01, error:  1.9625109998e-01
   Time: 1.902e+01, norm:  8.1592005489e-01, error:  1.9836917542e-01
   Time: 1.911e+01, norm:  8.1461149493e-01, error:  2.0023973560e-01
   Time: 1.922e+01, norm:  8.1372343236e-01, error:  2.0117397025e-01
   Time: 1.931e+01, norm:  8.1356992881e-01, error:  2.0086007326e-01
   Time: 1.942e+01, norm:  8.1292601530e-01, error:  2.0115038554e-01
   Time: 1.951e+01, norm:  8.1178141415e-01, error:  2.0261373541e-01
   Time: 1.962e+01, norm:  8.1021029339e-01, error:  2.0476704421e-01
   Time: 1.971e+01, norm:  8.0967605036e-01, error:  2.0523460118e-01
   Time: 1.983e+01, norm:  8.0955854426e-01, error:  2.0465194093e-01
   Time: 1.991e+01, norm:  8.0912943594e-01, error:  2.0475097414e-01
   Time: 2.000e+01, norm:  8.0810744660e-01, error:  2.0596819438e-01
